pages/4_Compare_Cases.py

```diff
@@ -0,0 +1,30 @@
+import streamlit as st
```

Comment:
Thinking about this a bit more, can you move this page to before "Understand the model"? That way the first three pages have local explanations, and the last page has a global explanation